### PR TITLE
chore: cache rust artifacts

### PR DESCRIPTION
**Pull request by zeapoz**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 /target
-/Cargo.lock
```
Comment (Rqnsom) [inline .gitignore:2]: can we keep this for normal non-CI users?

**Review by Rqnsom — APPROVED** [review]
LGTM

**Review by kylegranger — APPROVED** [review]
👍

File: .gitignore (hunk 1 above)
Comment (zeapoz) [reply .gitignore:2]: I guess we could, any particular reason for keeping it?

**Pull request merged**